pkg/util/testUtil.go

```diff
@@ -30,16 +30,17 @@ import (
 )
 
 // RunTemplate renders a yaml template string in the yaml_configs.go file
-func RunTemplate(tmpl string, input interface{}) string {
+func RunTemplate(tmpl string, input interface{}) (string, error) {
```

Comment:
no need to return error here, because this function will never actually return it

Reply:
Ok